genomedata/__init__.py

```diff
@@ -335,10 +307,9 @@ def add_track_continuous(self, trackname):
             raise NotImplementedError("""Adding tracks is only supported \
```

Comment:
Keep a note of this, when you add bigWig you're going to need a sensible error here too.